massgov/mayflower-patternlab
# DP -825 board org accordion

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

This PR adds a new organism 'teaser listing group' to wrap multiple teaser listings in a single accordion behavior.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-8825)


## Steps to Test


1. View http://localhost:3000/?p=pages-organization-boards and confirm that the first 8 items show, and the rest are behind an accordion.
2. View http://localhost:3000/?p=organisms-teaser-listing-group-showing-few-items and confirm that only 4 items are show, with the rest behind the accordion



#### Impacted Areas in Application


* ADDED:  @organisms/by-author/teaser-listing-group
